### PR TITLE
Changes by create-pull-request action

### PR DESCRIPTION
**Pull request by nk-coding**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: wishlist.graphql [hunk 1]
```diff
@@ -239,46 +248,6 @@ type WishlistConnection @shareable {
 	totalCount: Int!
 }
 
-"""
-Describes the fields that a wishlist can be ordered by.
-"""
-enum WishlistOrderField {
```
Comment (nk-coding) [inline wishlist.graphql:245]: why was this removed and instead the common one used?

File: wishlist.graphql [hunk 2]
```diff
@@ -123,22 +123,14 @@ type ProductVariantConnection @shareable {
...
+	Retrieve user with wishlists.
 	"""
-	wishlists(
-		"""
-		Describes that the `first` N wishlists should be retrieved.
-		"""
-		first: Int,
-		"""
-		Describes how many wishlists should be skipped at the beginning.
+	user(
```
Comment (nk-coding) [inline wishlist.graphql:128]: and why is there a user query?
you should have a resolver, but not the query

Comment (delvh) [reply wishlist.graphql:128]: So, what do we do with this PR then?
